dev/make-fast-fail.py

```
@@ -0,0 +1,66 @@
+                print(f"✅ Completed {target}", flush=True)
+                del alive[proc]
+                continue
+
+            print(f"❌ Failed: {target}", flush=True)
+            for other in list(alive.keys()):
+                if other is proc:
+                    continue
+                try:
+                    other.terminate()
+                except Exception:
+                    pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)